src/main/java/com/homework/lesson10/ValidatorsApp.java

```diff
@@ -0,0 +1,32 @@
+package com.homework.lesson10;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class ValidatorsApp {
+
+    public boolean isPhoneNumber(String number) {
+        Pattern pattern = Pattern.compile("^\\\\+?[0-9. ()-]{10,25}$");
```

Comment:
Please use constants for the regulars. 
Example:
`private static final String PHONE_NUMBER_REQULAR_EXPRESSION="^\\\\+?[0-9. ()-]{10,25}$"; `